cmd/monaco/account/delete.go

```diff
@@ -103,19 +103,15 @@ func loadManifest(fs afero.Fs, manifestName string) (manifest.Manifest, error) {
 	}
 	// Sanitize manifest file path to manifest yaml file
 	manifestName = filepath.Clean(manifestName)
-	absManifestFilePath, err := filepath.Abs(manifestName)
```

Comment:
this just causes problems in test context with e.g. afero.
I've deleted it, as i think (?) it is not really necessary, please confirm